core/io/resource_loader.cpp

```diff
@@ -359,6 +359,18 @@ void ResourceLoader::_thread_load_function(void *p_userdata) {
 		if (_loaded_callback) {
 			_loaded_callback(load_task.resource, load_task.local_path);
 		}
+	} else if (load_task.cache_mode != ResourceFormatLoader::CACHE_MODE_IGNORE) {
+		Ref<Resource> existing = ResourceCache::get_ref(load_task.local_path);
+		if (existing.is_valid()) {
+			//referencing is fine
```

Comment:
```suggestion
			// Referencing is fine.
```
Not sure what the comment is about.

Reply:
I think this accidentally get moved over when copy-pasting the code from elsewhere. I'll remove it.